Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/CallbackRequest.hs

```diff
@@ -15,8 +15,55 @@
 module Storage.Queries.CallbackRequest where
 
 import Domain.Types.CallbackRequest
+import Domain.Types.Merchant as DM
+import Kernel.External.Encryption
+import Kernel.Prelude
 import Kernel.Storage.Esqueleto as Esq
-import Storage.Tabular.CallbackRequest ()
+import Kernel.Types.Id
+import Kernel.Types.Time
+import Storage.Tabular.CallbackRequest
 
 create :: CallbackRequest -> SqlDB ()
 create = Esq.create
+
+findAllCallbackRequest :: Transactionable m => Id DM.Merchant -> Maybe Int -> Maybe Int -> m [CallbackRequest]
```

Comment:
`findAllCallbackRequest merchantId mbLimit mbOffset `is the same as
`fetchCallbackRequest merchantId mbLimit mbOffset Nothing Nothing Nothing`